manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://url.3u.com/zmAJjyaa
+  InstallerSha256: 230A196E56C37F9B1AB3451A27D4CF435202741A35E5E805622628E2274011ED
+  InstallerSwitches:
+    Silent: 0
+    SilentWithProgress: 0

```

Comment:
These switches do not work to install the application silently
![image](https://user-images.githubusercontent.com/12611259/193650929-93f037ca-5e0e-4f9c-bd34-d1a924e39de2.png)